baggage/privacy.md

```diff
@@ -7,7 +7,8 @@ This standard makes it possible for systems to operate on a known, standardized
 Systems MUST assess the risk of header abuse. This section provides some considerations and initial assessment of the risk associated with storing and propagating this header. Systems may choose to inspect and remove sensitive information from the fields before processing or propagating the received data. All mutations should, however, conform to the list of mutations defined in this specification.
 
 ## Privacy of the baggage header
+
 The main purpose of this header is to provide additional system-specific information to other systems within the same trust-boundary.
-The `baggage` header may contain any <a href="#opaque">opaque</a> value in any of the keys.
+The `baggage` header may contain any <a>opaque</a> value in any of the keys.
```

Comment:
the link is actually `#dfn-opaque`. But without any `href` it works as well based on the definition word.